packages/common/src/authentication/api/hooks/useUserDetails.ts

```diff
@@ -14,5 +14,8 @@ export const useUserDetails = () => {
 
 export const useUserStores = (token: string) => {
   const api = useAuthApi();
-  return useQuery(api.keys.me(token), api.get.stores(), { cacheTime: 0 });
+  return useQuery(api.keys.me(token), api.get.stores(), {
```

Comment:
Just because it is daft to ask for `me` when you have no token 🤷  
Actually not really required now that the underlying issue is fixed